ciphey/basemods/Decoders/URL.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional, Dict, List
+from urllib.parse import unquote
+
+from ciphey.iface import ParamSpec, Config, T, U, Decoder, registry
+
+
+@registry.register_multi((str, str), (bytes, bytes))
+class URL(Decoder[T, U]):
+    def decode(self, ctext: T) -> Optional[U]:
+        """Write the code that decodes here
```

Comment:
Can I also get a descriptive docs here? What does your code actually do? This is for the future so people don't have to guess :)

Reply:
Sure, i changed the comment